.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,62 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
```

Comment:
I guess you need to add the label ``triage`` in your list https://github.com/ITISFoundation/IXI025/labels

Reply:
ok, thanks for pointing that out